### ISSUE TITLE
Several problems with packages when trying to compile the book

### ISSUE THREAD

**Issue by martynabogacz** [afrimapr/afrimapr-book#20]
Hi I keep getting errors with packages and consequently the book doesn"t compile. I tried to solve it but was unsuccessful. In particular, there is problem with deployment as no afrilearnr package is found:
https://github.com/afrimapr/afrimapr-book/runs/2280581394?check_suite_focus=true#step:21:1546

Moreover, there is a problem with afrihealthsites:
https://github.com/afrimapr/afrimapr-book/runs/2280581426#step:4:675

I would appreciate if someone could look into it. 

Many thanks

**Comment by Robinlovelace** [comment]
I suggest trying again now that binary versions of `lwgeom` are available for Mac.

**Comment by andysouth** [comment]
Thanks both. I'm having a look now.
The first error seems to be caused by afrihealthsites & sf. Will see what I can do. 

Error: (converted from warning) package ‘sf’ was built under R version 4.0.4
Execution halted
ERROR: lazy loading failed for package ‘afrihealthsites’

**Comment by andysouth** [comment]
I didn't mean to close that with the commit - lets see if it works first ;-)

**Comment by andysouth** [comment]
remaining error : 
Error: Error: A step failed in stage "deploy": deploy.
<error/rlang_error>
In addition: Warning messages:
path for html_dependency not found: 
1: In formatC(x = c(1970L, 1980L, 1990L, 2000L, 2010L, 2020L), digits = 0,  :
  coercing argument to "character" for format="s"
2: In formatC(x = c(1970L, 1980L, 1990L, 2000L, 2010L, 2020L), digits = 0,  :
  coercing argument to "character" for format="s"
3: Values have found that are less than the lowest break 
4: Values have found that are higher than the highest break

**Comment by martynabogacz** [comment]
Let me have a look at this. Thanks a lot Andy.

Martyna
________________________________

**Comment by andysouth** [comment]
Looking good @martynabogacz 👍  

Seems like you've fixed it ?

**Comment by martynabogacz** [comment]
There is some progress. I managed to uncomment whole ch.5 where I use tmap for creating static and interactive maps. I fixed the bugs that were in the code. But there is one more problem remaining, when I try to uncomment mapview-related code. The error shows some issues with  missing "html_dependency":
https://github.com/afrimapr/afrimapr-book/runs/2298605950#step:21:1931

I have been reading about this, and it seems that it is a common issue. Some suggest downgrading the package but it didn't work for me.

Any suggstions are welcome :-)
